src/AxiomV2Addresses.sol

```diff
@@ -58,6 +58,25 @@ library AxiomV2Addresses {
         }
     }
 
+    /// @notice Returns the address of the cross-chain version of the AxiomV2Query contract on the chain corresponding to `chainId`
+    /// @param targetChainId The chainId of the AxiomV2Query contract
+    /// @param sourceChainId The chainId of the source chain
+    /// @return addr The address of the AxiomV2Query contract
+    function axiomV2QueryCrosschainAddress(uint64 targetChainId, uint64 sourceChainId)
```

Comment:
this is probably an important convention that we should figure out and set for the entire team now in terms of variable ordering (since it could lead to some unfortunate and inadvertent mistakes downstream). currently I've had a source chain, target chain convention in my head. however, I can see how making the target chain first makes a lot of sense as well. is one preferred over the other?

Reply:
I don't have a strong opinion here -- chose this order here so that the first argument is the chain the contract lives on so that the argument list is an extension of the one for `axiomV2QueryAddress`.  In the case of the SDK, are the arguments always named?